### ISSUE TITLE
Improve progress reporting for pack download/extraction

### ISSUE THREAD

**Issue by matthewrsj** [clearlinux/swupd-client#255]
Pack download/extraction progress reporting is sub-par and currently just makes the output more verbose without actually giving the user better information. The following format should be achieved
```
Downloading 3 packs...
[1/3] Downloading pack os-core for version 17200...
[1/3] Extracting pack...
[2/3] Downloading pack test-bundle for version 17110...
[2/3] Extracting pack...
[3/3] Downloading pack test-bundle-2 for version 17120...
[3/3] Extracting pack...
```
The problem is that we don't know if those packs exist until we try to download them.

Some (failing) work can be found on https://github.com/matthewrsj/swupd-client/tree/improve-progress

**Comment by unixbhaskar** [comment]
Any progress?

**Comment by matthewrsj** [comment]
Not on this particular issue, no.

**Comment by castulo** [comment]
@otaviobp  with the current way we are reporting progress, I don't think this issue is valid anymore. We are no longer showing the extraction of the packages, just the download progress. The extraction is not too significant since it is done in parallel with the download, so it only takes a few seconds once the download has finished.

This is the way output is being shown now for content downloading/extraction: 
```
Downloading packs (443.70 Mb) for:
 - ipmitool
 - lib-imageformat
 - qt-basic
 - fonts-basic
 - python3-basic
 - libX11client
 - lib-opengl
 - lib-openssl
 - p11-kit
 - findutils
 - qt-core
 - libglib
 - libstdcpp
 - os-core
 - xterm
 - ansible
 - blender
	...100%
Verifying files
	...100%
No extra files need to be downloaded
```

**Comment by castulo** [comment]
I think this issue can be closed now. If there is something else necessary for this feel free to open a new issue for that specific need.
